### PR TITLE
add ability to add a branding-stylesheet next to the default one 

### PR DESCRIPTION
**Pull request by Jessman5**
To modify the styleguide to fit to the brand.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,9 +86,10 @@ There are a bunch of options you can use:
 - `inputPagesPath`: The input pages path.
 - `outputPagesPath`: The output pages path.
 
-- `templateStyleguideStylesheet`: The stylesheets to include.
+- `templateStyleguideStylesheet`: The basic stylesheets of the styleguide (can be overwritten, not recommended).
+- `templateStyleguideBrandingStylesheet`: The additional stylesheets to include for customizing the whole theme.
```
Comment (mariusbuescher) [inline README.md]: Pleas do not call it `templateStyleguideBrandingStylesheet`. There are several reasons:

1. Not always useful for branding
2. Can be more than one

For me a better name would be `templateGlobalStylesheets`

File: generate/render-full-template.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ function renderFullTemplate( options, env, navTree, fileContents, addOptions ) {
         outputPagesPath: options.setup.outputPagesPath,
 
         templateStyleguideStylesheet: options.setup.templateStyleguideStylesheet,
+        templateStyleguideBrandingStylesheet: options.setup.templateStyleguideBrandingStylesheet,
```
Comment (mariusbuescher) [inline generate/render-full-template.js]: See above

File: src/templates/elements/head.njk [hunk 3]
```diff
@@ -9,6 +9,11 @@
 {# Style Guide Styles #}
 <link rel="stylesheet" href="{{ wayToRoot }}{{ webPath }}css/sg-style.css">
 
+{# Styleguide Branding Styles #}
+{% for styles in templateStyleguideBrandingStylesheet %}
+    <link rel="stylesheet" href="{{ styles }}" />
```
Comment (mariusbuescher) [inline src/templates/elements/head.njk]: Missing `type` attribute.

File: src/templates/reduced.njk [hunk 4]
```diff
@@ -18,7 +18,7 @@
             {% import "single.njk" as singleView %}
             {{ singleView.render(
             element,
-            webPath=webPath,
+            webPath = webPath,
```
Comment (mariusbuescher) [inline src/templates/reduced.njk:21]: Looks better, thanks.

**Review by mariusbuescher — CHANGES_REQUESTED** [review]

**Comment by Jessman5** [comment]
all done.

**Review by mariusbuescher — APPROVED** [review]

**Pull request merged**